spec/factories/legal_aid_applications.rb

```diff
@@ -320,6 +320,16 @@
       own_home { 'mortgage' }
     end
 
+    # trait :with_no_increased_cost_limit do
```

Comment:
Are these purposefully commented out? I don't see why it needs to be commented out

Reply:
I created them and then didn't use them. I will remove them, they can be recreated if required in the future.